nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/model/impl/AbstractProperty.java

```diff
@@ -65,6 +65,13 @@
      */
     public static final int IS_SECURED = 64;
 
+    /**
+     * Whether or not this property is deprecated.
```

Comment:
```suggestion
     * Whetherthis property is deprecated.
```
?

Reply:
Won't do this change because there's several occurrences of such sentences. Or do you want me to update them too?